schemas/apiDefinition.swagger.schema.json

```diff
@@ -1259,6 +1264,9 @@
         "x-ms-dynamic-values": {
           "$ref": "#/definitions/x-ms-dynamic-values"
         },
+        "x-ms-enum": {
```

Comment:
Is it supported extension? I am not aware of it in Power Automate, if it is for Power Apps then it is strange why it is a new extension instead of existing.
Can we either specify only supported custom extensions or allow all of them at once like 'x-ms-*' if possible. I do not think we should explicitly specify all custom extensions that we see in swaggers.

Reply:
While this extension doesn't have any impact on our platform currently, it is common enough for swaggers that are generated from tooling (i.e. AutoRest). My goal with this swagger is to enable usage within VS Code's 'Problems' window to be increasingly helpful to show only actual problems.